starknet-core/src/types/transaction_request.rs

```diff
@@ -30,12 +23,19 @@ pub enum AddTransactionResultCode {
     TransactionReceived,
 }
 
+#[derive(Debug, Serialize)]
```

Comment:
Looks like all the changes to `transaction_request.rs` are just re-ordering things. Any specific reason for doing that?

Reply:
Ah, I just reordered them alphabetically, when I was exploring the above mentioned types between `transaction.rs` and `transaction_request.rs`. Was easier to visually compare that way.